keps/sig-apps/3998-job-success-completion-policy/kep.yaml

```diff
@@ -5,7 +5,7 @@ authors:
 owning-sig: sig-apps
 participating-sigs:
   - sig-apps
-status: implementable
+status: implemented
```

Comment:
Don't change this, yet. If this slips 1.33 this will be misleading. We usually update this field after the release, while closing the issue tracking a particular feature.

Reply:
Addressed in 9585ac67db8477712e382122c47a9d1d8df05dfd